Header header
float32 temperature_f

